templates/quickstart-jira-dc.template.yaml

```diff
@@ -426,39 +418,40 @@ Parameters:
     Default: 60000
     Description: The number of milliseconds to sleep between runs of the idle object eviction thread. When non-positive, no idle object eviction thread will be run
     Type: String
-  DeployEnvironment:
-    Default: prod
-    AllowedValues:
-      - prod
-      - stg
-      - dr
-    ConstraintDescription: Must be either Production (prod), Staging (stg), or Disaster Recovery (dr).
-    Description: Environment of the application - either Production, Staging, or Disaster Recovery (DR)
+  DeploymentAutomationRepository:
+    Default: 'https://bitbucket.org/atlassian/dc-deployments-automation.git'
+    Type: String
+    Description: The deployment automation repository to use for per-node initialisation. Leave this as default unless you have customisations.
+  DeploymentAutomationBranch:
+    Default: "master"
     Type: String
+    Description: The deployment automation repository branch to pull from.
+  DeploymentAutomationPlaybook:
+    Default: "aws_jira_dc_node.yml"
+    Type: String
+    Description: The Ansible playbook to invoke to initialise the Jira node on first start.
+  DeploymentAutomationKeyName:
+    Default: ""
+    Type: String
+    Description: Named KeyPair name to use with this repository. The key should be imported into the SSM parameter store. (Optional)
   HostedZone:
     Default: ''
     ConstraintDescription: Must be the name of an existing Route53 Hosted Zone.
     Description: The domain name of the Route53 PRIVATE Hosted Zone in which to create cnames
     Type: String
-  JiraDownloadUrl:
```

Comment:
I am a bit worried about how Jira guys are going to test their unreleased Jira versions. I have done some investigation into their builds but I cannot see how they are passing different versions. The build is here: https://server-gdn-bamboo.internal.atlassian.com/browse/JIRA-ATJDCDT
and I have asked the Delta team to provide additional explanation of how they are testing QS.

Reply:
I think there is some tension here between developers and customers. This is definitely a feature that is useful when developing with quickstart but I think that adding unnecessary parameters does make it a bit more painful for customers using the quickstart for deploying their production instances. Is there a reason ordinary customers would want a custom version of Jira?

Reply:
It depends how we are downloading the new versions but AFAIK for Confluence (maybe Jira is the same) we are pushing the milestones, EAPs and betas to the public S3 bucket.

@bpar476 I agree with you, this should be useful internally not externally. There are some cases when support might to provide a custom version related to a support ticket for them but I am not sure if this is not sent as zip file through the support ticket.